pkg/loop/plugin_median.go

```diff
@@ -17,12 +17,14 @@ import (
 	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
 	pb "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb"
 	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+	"github.com/smartcontractkit/chainlink-relay/pkg/utils"
 )
 
 // PluginMedianName is the name for [PluginMedian]/[NewGRPCPluginMedian].
 const PluginMedianName = "median"
 
 type PluginMedian interface {
```

Comment:
Is this better as PluginMedianService MedianPluginServicer? i like the latter -- the `er` makes it clear that it's an interface and `MedianPluginServicer` expresses the intention of a service for median plugins

Reply:
Hmm I think having `Service` and `Servicer` would be confusing... :thinking:

Reply:
I'm actually not sure what you mean here. This is not a service _for_ median plugins, per se - the plugin _is_ a service. Is it more clear with s/`NewMedianPluginFactory`/`NewMedianFactory`?

Reply:
yeah better